packages/rancher-operator-crd/package.yaml

```diff
@@ -1,2 +1 @@
 url: https://github.com/rancher/rancher-operator/releases/download/v0.1.4/rancher-operator-crd-0.1.4.tgz
```

Comment:
Patch this to v1.0.0 since we have already released v0.1.4 in https://github.com/rancher/charts/tree/release-v2.5/charts/rancher-operator/rancher-operator.

Same with original chart.

Use `version` in the package.yaml.

Reply:
fixed.